gulpfile.js

```diff
@@ -2,6 +2,7 @@ var gulp = require('gulp');
 var concat = require('gulp-concat');
 var del = require('del');
 var Server = require('karma').Server;
+const babel = require('gulp-babel');
```

Comment:
I see why you created a `const` here, but for purposes of consistency, don't you think that the other `requires` should also be of `const`?